models/evc.py

```diff
@@ -117,6 +117,9 @@ def __init__(self, controller, **kwargs):
         )
         self.service_level = kwargs.get("service_level", 0)
         self.circuit_scheduler = kwargs.get("circuit_scheduler", [])
+        self.flow_removed_at = get_time(kwargs.get("flow_removed_at")) or None
+        self.updated_at = get_time(kwargs.get("updated_at")) or now()
+        self.execution_rounds = get_time(kwargs.get("execution_rounds")) or 0
```

Comment:
`get_time` on `execution_rounds` doesn't look right to me. Can you double check this line?

Reply:
Fixed see [link](https://github.com/kytos-ng/mef_eline/blob/2622aac8b1de25ef80df03a9d3e2dfec9c5877dd/models/evc.py#L122).